twisted/conch/scripts/ckeygen.py

```diff
@@ -33,7 +33,8 @@ class GeneralOptions(usage.Options):
                      ['type', 't', None, 'Specify type of key to create.'],
                      ['comment', 'C', None, 'Provide new comment.'],
                      ['newpass', 'N', None, 'Provide new passphrase.'],
-                     ['pass', 'P', None, 'Provide old passphrase.']]
+                     ['pass', 'P', None, 'Provide old passphrase.'],
+                     ['format', 'o', 'sha256-base64', 'Fingerprint format of key file.']]
 
```

Comment:
I could have sworn I did this already, but, here it is again: 

This is adding stuff. You're always allowed to add stuff.  Is there some attribute or function being deleted here that I'm not seeing?